Multisig/Cross-layer/Analytics/TrackingEvent.swift

```diff
@@ -47,10 +47,13 @@ enum TrackingEvent: String, Trackable {
     case transactionsHistory                        = "screen_transactions_history"
     case transactionsNoSafe                         = "screen_transactions_no_safe"
     case transactionsDetails                        = "screen_transactions_details"
+    case transactionDetailsRejectionConfirmation    = "transactions_rejection_confirmation"
```

Comment:
The "screen_" prefix is missing in the event name

Reply:
true